src/Orleans/Statistics/IPerformanceMetrics.cs

```diff
@@ -10,34 +10,89 @@ namespace Orleans.Runtime
 {
     public interface ICorePerformanceMetrics
     {
+        /// <summary>
+        /// CPU utilization
+        /// </summary>
         float CpuUsage { get; }
+        /// <summary>
```

Comment:
This is a nit and I don't feel too strong about this (due to working with VS), but adding empty between the comment and previous line adds clarity to many. I don't know if there are anything rule-like in Orleans codebase for this.